<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfPwd</class>
 <widget class="QWidget" name="ConfPwd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ConfPwd{
background-color: rgb(255, 255, 255);
}
QToolButton{
font: 20pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(0, 122, 165);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(255, 255, 255);
}

QLabel{
font: 25pt &quot;文泉驿等宽微米黑&quot;;
color:rgb(0, 122, 165);
}
QLineEdit{
font: 25pt &quot;文泉驿等宽微米黑&quot;;
color:rgb(15, 41, 68);
border-radius:5px;
border:2px solid rgb(0, 122, 165);
background-color: rgb(255, 255, 255);
}
QRadioButton{
font: 18pt &quot;文泉驿等宽微米黑&quot;;
color:rgb(0, 122, 165);
background-color: rgb(255, 255, 255);
}
QRadioButton::indicator{
width:25px;
height:25px;
margin-left:0px;
}
QRadioButton::indicator::unchecked {
image: url(:/Image/disenable.png);
}
QRadioButton::indicator::checked {
image: url(:/Image/enable.png);
}
QGroupBox{
font: 20pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(0, 122, 165);
border:2px solid rgb(0, 122, 165);
border-radius:8px;
margin-top:22px;
}
QGroupBox:title{
subcontrol-origin: margin;
left: 5px;
}</string>
  </property>
  <widget class="QLabel" name="lb_newPasswd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
     <pointsize>25</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>新密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOldPwd">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
     <pointsize>25</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>1-6位数字</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNewPwd">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
     <pointsize>25</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>1-6位数字</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnPwdOk">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>80</y>
     <width>110</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>更新
密码</string>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnResetPwdOk">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>80</y>
     <width>110</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>恢复
密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_oldPasswd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
     <pointsize>25</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>原密码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 122, 165);</string>
   </property>
   <property name="text">
    <string> 密码设置</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnSuper">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>150</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>超级用户</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnAdmin">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>80</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>管理员</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>1024</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
font: 22pt &quot;文泉驿等宽微米黑&quot;;
color:rgb(0, 122, 165);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(255, 255, 255);
}</string>
   </property>
   <widget class="QToolButton" name="tBtn_8">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>15</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_del">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>15</y>
      <width>110</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_0">
    <property name="geometry">
     <rect>
      <x>815</x>
      <y>15</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_7">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>15</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_5">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>15</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_4">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>15</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_9">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>15</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_1">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>15</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
font: 20pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(0, 122, 165);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_6">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>15</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_2">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>15</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_3">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>15</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
